### ISSUE TITLE
Define unwanted data categories in datasets

### ISSUE THREAD

**Issue by deanwampler** [The-AI-Alliance/open-trusted-data-initiative#230]
### Discussed in https://github.com/The-AI-Alliance/open-trusted-data-initiative/discussions/226

<div type='discussions-op-text'>

<sup>Originally posted by **deanwampler** February 19, 2026</sup>
There is a discussion in the now-archived `agents-wg` project about "unwanted data catagories in datasets": https://github.com/The-AI-Alliance/agents-wg/discussions/29. This discussion copies over that content, as it's a relevant topic for OTDI.

@billstout-snc posted the following to start the discussion:

Suggestion: **Define categories of unwanted data in datasets, declare those categories in dataset cards**

The AI supply chain starts with data. Datasets are a collection of assets and liabilities. Assets include information useful for a model, and there's debate on if liabilities are useful for a model to know and recognize, or if a model should not have been trained on unwanted data.

Example categories of potentially unwanted data listed below:

| Category | Description of the Problem | Impacted Party | Evidence (Links) |
| :------- | :------------------------- | :------------- | :--------------- |
| Personal Identifiable Information (PII) |	Can violate privacy laws (e.g., GDPR/CCPA) if scraped or shared without consent; risk of regulatory fines and reputational damage. | Producer & Deployer | [Clearview AI scraping case](https://archive.is/Ok6gR) |
| Sensitive Health Data | Sharing or using patient data without proper consent can violate HIPAA and other healthcare privacy regulations; risk of lawsuits and heavy fines. | Producer & Deployer | [Google’s “Project Nightingale”](https://archive.is/bu3mq) |
| Copyrighted or Licensed Content | Ingesting protected text or images without permission can lead to infringement claims; potential lawsuits, takedowns, or cease-and-desist orders. | Producer & Deployer | [Getty Images lawsuit vs. Stability AI](https://www.penningtonslaw.com/news-publications/latest-news/2024/generative-ai-in-the-courts-getty-images-v-stability-ai) |
| Defamatory or Misinformation | Models trained on false or defamatory data may reproduce harmful statements, exposing deployers to defamation claims; can erode trust in AI systems. | Producer & Deployer | [OpenAI sued for defamation over ChatGPT “hallucinations”](https://www.forbes.com/sites/siladityaray/2023/06/08/openai-sued-for-defamation-after-chatgpt-generates-fake-complaint-accusing-man-of-embezzlement/) |
| Hate Speech or Extremist Content | Risk of amplifying hateful or violent ideologies; can damage brand reputation and invite regulatory scrutiny. | Deployer | [YouTube algorithm controversy](https://www.ucdavis.edu/curiosity/news/youtube-video-recommendations-lead-more-extremist-content-right-leaning-users-researchers) |
| Unlabeled / Poorly Labeled Data | Incorrect, biased, or offensive labels can propagate errors, biases, or harmful outcomes in downstream models; may require costly dataset rework. | Producer | [MIT’s withdrawal of “80 Million Tiny Images” dataset](https://venturebeat.com/ai/mit-takes-down-80-million-tiny-images-data-set-due-to-racist-and-offensive-content/) |
| Malware or Malicious Code | Trojans or backdoors hidden in training data or dependencies can compromise models, infrastructure, or end-user systems. | Producer | [PyTorch-nightly dependency compromise](https://pytorch.org/blog/compromised-nightly-dependency/) |
| Regionally Restricted or Illegal Content | Material that is banned or heavily regulated in certain countries (e.g., Nazi symbols in Germany, certain religious or political content in other jurisdictions). Violations can result in fines, local bans, or legal actions. | Producer & Deployer | [Germany’s ban on Nazi symbols in games](https://www.bbc.com/news/technology-45149304) |
| Child Sexual Exploitation (CSE) Material | Universally illegal. Even inadvertent inclusion in datasets can lead to severe criminal penalties; immediate takedown notices, public outcry, and possible prosecution. | Producer & Deployer | [Platform struggles with CSE detection](https://www.arxiv.org/pdf/2503.00433) |
| Locally Censored or Regulated Content | Many regions impose censorship on specific political, religious, or cultural information (e.g., speech critical of government, certain religious references). Failing to comply can lead to shutdowns or blocking within a region. | Deployer | [Google facilitated Russia and China’s censorship requests](https://www.theguardian.com/world/2025/feb/15/google-helped-facilitate-russia-china-censorship-requests) (coverage of broader content restrictions) |

@julietshen replied with this comment:

A few good sources to cross-reference definitions and categories with include the TSPA (trust and safety professional association)'s abuse types page: https://www.tspa.org/curriculum/ts-fundamentals/policy/abuse-types/.

CSE, for example, may have different degrees of feasibility depending on if it's text or multimodal. (not a lawyer and based on my best guess, I'd recommend checking with a lawyer here) Grooming text is not necessarily treated the same way as CSAM media and thus may be somewhat easier to detect without having to get licenses to access databases of known CSAM hashes.

@roostorg is working on some tooling to make hash matching against known illegal content easier, building on the work done in https://github.com/facebook/ThreatExchange/tree/main/hasher-matcher-actioner.

And then, of course, the need to reflect these categories of unwanted data in a global perspective and ensure any filtering is effective for non-english content.</div>

**Comment by julietshen** [comment]
Adding on with the latest updates from @roostorg, Coop is available, open source (Apache 2), and used in production by a few organizations. It's integrated with HMA and provides hash matching against 3rd party hash databases for known CSAM, terrorism and violent extremism content (TVEC), and non-consensual intimate imagery. It's also integrated with NCMEC's CyberTip Report API so Child Sexual Exploitation (CSE) Material can be reported to the right authorities. In addition to hash matching, it comes with customizable integrations to API-based filters. We're aiming to make it easier to use open weight models with Coop as well, which opens up the ability to filter out things like PII (ie. [Roblox](https://huggingface.co/Roblox/roblox-pii-classifier) and [OpenAI](https://huggingface.co/openai/privacy-filter) both have PII classifiers) and other types of unwanted data.

https://github.com/roostorg/coop

A screenshot of the UI to configure a hash bank connected to NCMEC (you still need to get your own credentials from NCMEC):

<img width="883" height="735" alt="Image" src="https://github.com/user-attachments/assets/276386ad-2897-46ad-96b2-c043a397c47a" />

<img width="1190" height="522" alt="Image" src="https://github.com/user-attachments/assets/7375588a-6eb7-433c-a1d9-20161adcedc8" />

<img width="1469" height="752" alt="Image" src="https://github.com/user-attachments/assets/df0a2f2a-ee87-40ac-940a-244e84bf7418" />
